staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -116,6 +124,7 @@ func InstallPathHandler(mux mux, path string, checks ...HealthzChecker) {
 	klog.V(5).Info("Installing healthz checkers:", formatQuoted(checkerNames(checks...)...))
 
 	mux.Handle(path, handleRootHealthz(checks...))
+	// TODO: let's remove this in a future version
```

Comment:
then we have to deprecate it, in code, the release notes and the docs.

Reply:
I'll leave this for the next release :)